### DISCUSSION TITLE
Resume_Review

### DISCUSSION THREAD

**Discussion by meetarora10** [recodehive/resume-pitch#94] [Q&A]
@sanjay-kv here is my resume. Kindly have a look.
[meetarora-12.pdf](https://github.com/user-attachments/files/17006477/meetarora-12.pdf)

**Comment by sanjay-kv** [comment]
[met.pdf](https://github.com/user-attachments/files/17089366/met.pdf)

[ATS Version of CV (24).docx](https://github.com/user-attachments/files/17089368/ATS.Version.of.CV.24.docx)

@meetarora10
